Fitness/Models/PersonalLink.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  File.swift
+//  Fitness
+//
+//  Created by Cameron Hamidi on 3/21/19.
+//  Copyright © 2019 Keivan Shahida. All rights reserved.
+//
+
+import Foundation
+
+struct PersonalLink {
+    enum PersonalLinkType {
```

Comment:
I think SiteType or Site might be a better name here. When using outside of PersonalLink, it will be namespaced as `PersonalLink.PersonalLinkType` which is quite redundant. `PersonalLink.Website` or something along those lines uses this namespacing better.

Reply:
Understood, I'll change it to `SiteType`